### PR TITLE
Integration of Onyx Protocol Token (XCN)

### PR DESCRIPTION
**Pull request by zasulskii**
Supported blockchains for XCN: Ethereum, BSC.
Native XCN bridge for swapping between Ethereum and BSC: bridge.onyx.org

### CODE REVIEW

**Comment by RanGojo** [comment]
Thanks for the PR @MerdyNumber1 . We will check it soon.

File: tokens/custom/xcn/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "name": "Onyx Protocol Token",
```
Comment (al-onyxprotocol) [inline tokens/custom/xcn/manifest.json]:
```suggestion
    "name": "Onyxcoin",
```

File: tokens/custom/xcn/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "symbol": "XCN",
+    "address": "0xA2cd3D43c775978A96BdBf12d733D5A1ED94fb18",
+    "decimals": 18,
+    "chainId": 1
+  }, {
+    "name": "Onyx Protocol Token",
```
Comment (al-onyxprotocol) [inline tokens/custom/xcn/manifest.json]:
```suggestion
    "name": "Onyxcoin",
```

**Review by al-onyxprotocol — CHANGES_REQUESTED** [review]

**Comment by al-onyxprotocol** [comment]
DEXes:

Ethereum:
[UniSwap v2 XCN/ETH](https://v2.info.uniswap.org/pair/0x859f7092f56c43bb48bb46de7119d9c799716cdf)
[UniSwap v3 XCN/ETH](https://info.uniswap.org/#/pools/0x5c7f114086ac472162b66b259eac89e820974461)

BNB Smart Chain:
[PancakeSwap v3 XCN/WBNB](https://pancakeswap.finance/info/v3/pairs/0xa694a7c24a84ec66db840caa745fce8413bde0d8)
[PancakeSwap v3 XCN/USDT](https://pancakeswap.finance/info/v3/pairs/0x1aaecb940a342d89258a297e2418b22ce1bf5d5e)
[AmpleSwap XCN/USDT](https://ampleswap.com/swap?outputCurrency=0x7324c7c0d95cebc73eea7e85cbaac0dbdf88a05b&inputCurrency=0x55d398326f99059fF775485246999027B3197955)

**Review by al-onyxprotocol — APPROVED** [review]

**Comment by RanGojo** [comment]
@MerdyNumber1 @al-onyxprotocol Thanks for the PR.
Your token will be updated in our production env in the next release during the next week.

**Pull request merged**